cpp/dolfinx/refinement/plaza.cpp

```diff
@@ -24,13 +24,23 @@ using namespace dolfinx::refinement;
 namespace
 {
 //-----------------------------------------------------------------------------
-// 2D version of subdivision allowing for uniform subdivision (flag)
+/// 2D version of subdivision allowing for uniform subdivision (flag)
+/// @param[in] indices Vector containing the
+/// global indices for the original vertices and potential new vertices at each
+/// edge. If an edge is not refined its corresponding entry is -1. Size
+/// `num_vertices + num_edges`
```

Comment:
```suggestion
/// @param[in] indices Vector of size `num_vertices + num_edges` containing new
/// global vertex indices for each point. If an edge is not refined its corresponding entry is -1.
/// 
```

Reply:
I changed this after feedback from Garth (https://github.com/FEniCS/dolfinx/pull/3010#discussion_r1462384505)